nodeup/pkg/model/cloudconfig.go

```diff
@@ -17,15 +17,22 @@ limitations under the License.
 package model
 
 import (
+	"bufio"
 	"fmt"
 	"k8s.io/kops/pkg/apis/kops"
+	"k8s.io/kops/pkg/apis/kops/util"
 	"k8s.io/kops/upup/pkg/fi"
 	"k8s.io/kops/upup/pkg/fi/nodeup/nodetasks"
+	"os"
 	"strings"
 )
 
 const CloudConfigFilePath = "/etc/kubernetes/cloud.config"
 
+// Required for vSphere CloudProvider
+const MinimumVersionForVMUUID = "1.5.3"
+const VM_UUID_FILE_PATH = "/root/vm_uuid"
```

Comment:
CloudInit will store the uuid at '/root/vm_uuid'. This will be then used by nodeup while creating cloud config file. The location '/etc/vmware/vm_uuid' looks better, will change it to this.